code/game/objects/items/stacks/sheets/glass.dm

```diff
@@ -358,7 +358,8 @@
 			var/obj/item/bodypart/O = H.get_bodypart(picked_def_zone)
 			if(!istype(O))
 				return
-			if(!H.shoes)
+			var/feetCover = (H.wear_suit && H.wear_suit.body_parts_covered & FEET) || (H.w_uniform && H.w_uniform.body_parts_covered & FEET)
+			if(!H.shoes && !feetCover)

```

Comment:
Actually I was thinking of generalizing to `is_bodypart_covered(FEET)` since mechanically speaking any equip-able item can have any of the `body_parts_covered` flags. But perhaps that's overkill?